src/ctapipe/io/simteleventsource.py

```diff
@@ -546,6 +546,14 @@ class SimTelEventSource(EventSource):
         ),
     ).tag(config=True)
 
+    subtract_pedestal_baseline = Bool(
+        True,
+        help=(
+            "Subtract the pedestal baseline from the waveform samples."
+            " This is done by default, but can be disabled for calibration events."
```

Comment:
This implementation seems to turn It off globally (for all events), not just for calibration events. Would be be better to really apply It only to calibration event types?  See the list of event types: https://github.com/cta-observatory/ctapipe/blob/7e40bd23019a06793b4ea2d5c29cbf1403093467/src/ctapipe/containers.py#L151-L172

Reply:
I think you need to turn it off globally: if you compute a new pedestal based on non-subtracted pedestal events, you also do not want to apply the initial pedestal subtraction of simtel to the physics events, right? You want to apply the newly computed pedestal, which will be absolute, not relative to the initial one.